python/ray/train/predictor.py

```diff
@@ -15,7 +15,8 @@
 
 try:
     import pyarrow
-    import pyarrow.Table as pa_table
+
+    pa_table = pyarrow.Table
```

Comment:
Just curious, does this change have any impact?

Reply:
Yes - in newer versions of pyarrow, `pyarrow.Table` is actually `pyarrow.lib.Table` and `import pyarrow.Table` fails:

```
>>> import pyarrow
>>> pyarrow.__version__
'7.0.0'
>>> import pyarrow.Table
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'pyarrow.Table'
```

whereas `pyarrow.Table` does exist and maps to the correct class:

```
>>> pyarrow.Table
<class 'pyarrow.lib.Table'>
```

Reply:
(i.e. all pyarrow tests failed for me locally)